portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,29 +19,54 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.Arrays;
 import java.util.ArrayList;
 import com.google.gson.Gson;
+import com.google.sps.servlets.Comment;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  ArrayList<Comment> commentHistory = new ArrayList<Comment>(); 
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    ArrayList<String> data = new ArrayList<String>();
-    data.add("Value 1");
-    data.add("Value 2");
-    data.add("Value 3");
-
-    String json = convertToJsonUsingGson(data);
+    if(commentHistory.isEmpty()){
+        response.setContentType("text/html");
+        response.getWriter().println("-1");
```

Comment:
-1 isn't a super helpful error message to display to the client. Something like "No Comments Found" might be more helpful.